src/fs/bootstrap_record.cc

```diff
@@ -0,0 +1,187 @@
+/*
+ * This file is open source software, licensed to you under the terms
+ * of the Apache License, Version 2.0 (the "License").  See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.  You may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/*
+ * Copyright (C) 2019 ScyllaDB
+ */
+
+#include "fs/bootstrap_record.hh"
+
+#include <boost/crc.hpp>
+#include <seastar/core/units.hh>
+
+namespace seastar::fs {
+
+namespace {
+
+constexpr unit_size_t alignment = 4 * KB;
+constexpr disk_offset_t boot_record_offset = 0;
+constexpr unit_size_t min_sector_size = 512;
+
+constexpr size_t aligned_boot_record_size =
+        ((sizeof(bootstrap_record_disk) / alignment) + (sizeof(bootstrap_record_disk) % alignment > 0)) * alignment;
```

Comment:
maybe:
```suggestion
        (1 + (sizeof(bootstrap_record_disk) - 1) / alignment) * alignment;
```
see: https://stackoverflow.com/a/2745086
This looks like quite common thing to do, maybe make a standalone function such as `round_up_to_multiple_of()`?